### PR TITLE
Digests input bucket configuration for the bot.

### PR DESCRIPTION
**Pull request by gnott**
Fixes https://github.com/elifesciences/digest-parser/issues/6

My best guess based on the pattern in the other bucket names, mostly modelled around the silent corrections bucket. Please feel free to add more commits to this branch if you see things to change directly.

### CODE REVIEW

File: projects/elife.yaml [hunk 1]
```diff
@@ -686,6 +687,8 @@ elife-bot:
                     deletion-policy: delete
                 "{instance}-elife-bot-sessions":
                     deletion-policy: delete
+                "{instance}-elife-bot-digests-input":
+                    deletion-policy: delete
```
Comment (giorgiosironi) [inline projects/elife.yaml]: this is needed so that `elife-bot-formula` builds don't leave buckets around after they have completed

**Comment by gnott** [comment]
I wasn't sure if S3 notifications from other buckets would be ignored, and on closer inspection the queue_worker may exit if it encounters one. I have some code changes in this PR I look forward to your review if you can please https://github.com/elifesciences/elife-bot/pull/717.

The other alternative was to attach this new bucket to a ``Ping`` workflow for now, but how it sends the ``data`` to the starter, it might not be compatible without additional changes. The "log it and continue" logic for the queue_worker is a more flexible improvement.

**Comment by giorgiosironi** [comment]
> wasn't sure if S3 notifications from other buckets would be ignored

I like this direction as a general pattern, let's work on the bot changes first.

**Comment by gnott** [comment]
The bot change to ``queue_worker.py`` is merged, and I'm deploying the change to ``prod`` now. This PR then I think is safe to merge, and it is approved, here goes!

**Comment by giorgiosironi** [comment]
Applying (a human has to do this) to the various environments.

**Comment by giorgiosironi** [comment]
- [ ] `ci`
- [ ] `end2end`
- [ ] `continuumtest`
- [ ] `prod`

**Comment by giorgiosironi** [comment]
Applied everywhere.

**Pull request merged**